### DISCUSSION TITLE
Strange things happen

### DISCUSSION THREAD

**Discussion by Hansvanwijhe** [flybywiresim/aircraft#3814] [Help]
Starting today, March 5, 2021, strange things happen within my FS2020. I am flying the FBW Airbus A32NX. When entering data by mouse clicks in the FMCU after a few mouse clicks FS2020 hangs. No input whatsoever (neither mouse, keyboard or joystick) is being accepted, the thing is just stone-dead. I have tried all FBW Airbus versions (Stable, Dev and CFBW) and with everyone of the three the same happens. (Using the latest versions of the FBW Airbuses). The hangs happen randomly, but most of the time when I am in the PERF page.  

Strangely enough (or perhaps not) this does not happen when I use the stock A320NEO.

Have any of you experienced the same? Or do any of you have an idea what I can do to circumnavigate this issue?

Thanks a lot,
Hans

**Comment by bmelen** [comment]
Hi Hans
I had the same problem a couple of weeks ago. I was adviced to emty my comunity folder exept for the FBW version in use.
Take a copy of the folder first.

Bert

**Reply by Hansvanwijhe** [reply]
Hi Bert,

Thank you very much for your help. After emptying the Community folder (except for the FBW A320), all worked well again. I only had 5 subfolders in my Community folder, i.e. Aerosoft's EDDK, EDLP and ENVA, ORBX's ENAL and Navigraph's navdata. After re-entering the subfolders one by one it seems that Navigraph's navdata is the culprit. I will reinstall Navigraph and if the problem persist, I will contact Navigraph. 
Again, thanks a lot for your help.

Hans

**Comment by Hansvanwijhe** [comment]
I did contact Navigraph about this issue and they inform me that the problem must be within FBW's Airbus. So here I am, stuck in between FBW and Navigraph :) What to do? For now I uninstalled Navigraph's navdata in order to be able to fly the A32NX.

Have any of you any idea or hint how I can resolve this issue?

Thanks,
Hans

**Comment by Hansvanwijhe** [comment]
It is getting weirder and weirder. The described issue only happens when I use an EDDK-LICJ flightplan, generated by SImbrief. Other flightplans I used didn’t cause any problem. You may want to try it, here’s the FP:

EDDK KUMI1C KUMIK Y854 BOMBI UN850 NATOR N850 ODINA LAGEN L50 ELB L12 GIANO GIAN4F LICJ

Hans
